wts/resources/oauth2.py

```diff
@@ -39,7 +43,7 @@ def find_valid_refresh_token(username):
     return has_valid
 
 
-def refresh_refresh_token(tokens):
+def refresh_refresh_token(tokens, idp):
```

Comment:
Same story here, recommend adding a unit test now that we've changed the behavior